### PR TITLE
adding Swedish and Bulgarian translations

### PR DESCRIPTION
**Pull request by ctlaltdieliet**
Added the Swedish and Bulgarian translation

### CODE REVIEW

File: i18n/i18n.jsx [hunk 1]
```diff
@@ -90,36 +80,60 @@ const languages = {
...
+        name: '中文 (繁體)',
+        order: 16,
+        url: zhTW,
+    },
+    ja: {
+        value: 'ja',
+        name: '日本語',
+        order: 17,
+        url: ja,
+    }
```
Comment (metanerd) [inline i18n/i18n.jsx:136]:
```suggestion
    },
```

**Review by metanerd — APPROVED** [review]
LGTM! Thank you @ctlaltdieliet

**Comment by jgilliam17** [comment]
Looks good overall
Noticed few minor issues
@metanerd Can we make changes to this PR or do we need to file followup tickets  to fix in the translation file?
1. Direct message in Swedish has () around the title `(Privat meddelande)`
Screen Shot 2021-01-11 at 10 25 41 AM

2. Playbooks and Incidents are missing translations, this may be expected for now as it's also missing for other existing languages. 
3. `Open a Direct message` text (from top + menu) flows out of menu modal in mobile view for both languages 
Screen Shot 2021-01-11 at 11 06 32 AM

**Comment by metanerd** [comment]
1. Let's do a followup for this. 
2. Missing translations in the plugins is to be expected. 
3. Interesting. I think we will inform translators of this and in the next language update these should be fixed.

**Comment by metanerd** [comment]
Thank you for your thorough review, @jgilliam17 !

**Review by jgilliam17 — APPROVED** [review]
Thank you @ctlaltdieliet and @metanerd 
Looks good to merge.
- Verified Swedish and Bulgarian translation; the issues mentioned in my previous comment are flagged for follow up with the translators and will be fixed in the next language update.

**Pull request merged**